blog/2023-03-15-implementing-an-english-auction-contract-in-solidity/implementing-an-english-auction-contract-in-solidity.md

```diff
@@ -0,0 +1,656 @@
+---
+title: Implementing an English Auction Contract in Solidity
+description: An English auction is ideal for selling NFTs because it gives all potential buyers a fair chance at placing bids until only one bidder with the highest bid is left.
+  - name: Oyeniyi Abiola Peace
```

Comment:
Hi @iamoracle your metadata here is missing the right structure "authors" specifically